nmspy/data/structs.py

```diff
@@ -739,19 +957,46 @@ class cGcTechnology(ctypes.Structure):
         ("mColour", common.Colour),
         ("miLevel", ctypes.c_int32),
         ("mbChargeable", ctypes.c_ubyte),
+        ("_padding0x235", ctypes.c_ubyte * 0x3),
         ("miChargeAmount", ctypes.c_int32),
         ("mChargeType", cGcRealitySubstanceCategory),
-        ("maChargeBy", common.cTkDynamicArray[ctypes.c_char * 0x10]),
+        ("maChargeBy", common.TkID[0x10]),
```

Comment:
This is an array though? `cTkDynamicArray<TkID<128> >` is the type in the game and this worked for me...